sale_financial_risk/models/res_config_settings.py

```diff
@@ -0,0 +1,10 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    include_done_orders = fields.Boolean(
```

Comment:
The name of this field is too generic.
Note that we are adding a field to the general class res.config.settings, so it would be convenient to refer to risk and sale.
We also try to avoid plurals.
Use include_risk_sale_order_done or something similar.